postinstall.py

```diff
@@ -0,0 +1,9 @@
+import shutil
+import platform
+import os
+
+if platform.system() == "Windows":
```

Comment:
I think you can move this script to a simply common js script instead. It doesn't really matter to be honest, but it would make more sense considering that this script will be executed from the node package manager

Reply:
makes sense